020_up_L22.py

```
@@ -0,0 +1,128 @@
+        print('---')
+        print('Click here to edit the information for this video:\n%s' % manageurl(f'video/{r["id"]}'))
+
+
+def repairer(code):
+    limit = uploader().MAX_BYTES
+
+    for file in tsfiles(code):
+        if path.getsize(file) > limit:
+            tmp = 'rep.%s' % file
+            # {fact rule=os-command-injection@v1.0 defects=1}
+            os.system(genrepair(file, tmp, limit * 8))
```

Ruff (0.14.5)

Starting a process with a shell, possible injection detected

(S605)